apps/public-reference/pages/_app.jsx

```diff
@@ -5,20 +5,28 @@ import "@bloom-housing/ui-components/styles/index.scss"
 import {
   addTranslation,
   UserProvider,
-  ConfigProvider, ApiClientProvider
+  ConfigProvider,
+  ApiClientProvider,
 } from "@bloom-housing/ui-components"
 import { headScript, bodyTopTag, pageChangeHandler } from "../src/customScripts"
 import { AppSubmissionContext, blankApplication } from "../lib/AppSubmissionContext"
-import { loadApplicationFromAutosave, loadSavedListing } from "../lib/ApplicationConductor"
+import ApplicationConductor, {
+  loadApplicationFromAutosave,
+  loadSavedListing,
+} from "../lib/ApplicationConductor"
 
 class MyApp extends App {
   constructor(props) {
     super(props)
 
     // Load autosaved listing application, if any
-    const autosavedApplication = loadApplicationFromAutosave()
+    const application = loadApplicationFromAutosave() || blankApplication()
     const savedListing = loadSavedListing()
-    this.state = { application: autosavedApplication || blankApplication(), listing: savedListing }
+    this.state = {
+      conductor: new ApplicationConductor(application, savedListing),
+      application: application,
+      listing: savedListing,
+    }
```

Comment:
Do we ever envision a use case where a user might have multiple applications in progress at the same time? This logic here sets up `application`, `conductor`, and `listing` as global singletons without scoping by ID or anything like that.

Since our routing doesn't support unique application IDs yet, this might just be something to punt on until later, but it seems to me like keeping this state in an object that's more easily scoped (vs. the root app class) might make it easier to support that use case down the line?

Reply:
I think we'd want to restrict users to having one application in progress for at least the short-medium term, more because I think the experience would be confusing to have multiples open at once, and we want to discourage half-completed applications. That said, I'm all in favor of scoping the variables underneath to give us more technical flexibility if we ever need it, so long as the overhead isn't too annoying.

Reply:
I was wondering about that as well when working on this update, but ending up with multiple applications getting autosaved and having to deal with all the mushrooming complexity of the data we're tracking across pages…oof, feels like a major headache.

Reply:
If it's a major headache, I'm fine with saying we're only going to allow one application at a time in-progress, so long as @slowbot is on board with that too.

Reply:
Cool, I’m totally fine punting on this then - we can tackle when/if it becomes relevant.